### PR TITLE
(BIDS-3256) Login and Signup pages - redesign

### PR DESCRIPTION
**Pull request by thib-wien**
This PR

- improves the designs of the login and registration pages,
- empties the header for these pages, only the logo remains.

### CODE REVIEW

File: frontend/pages/register.vue [hunk 1]
```diff
@@ -146,33 +138,32 @@ const canSubmit = computed(() => email.value && password.value && confirmPasswor
 <style lang="scss" scoped>
 @use "~/assets/css/fonts.scss";
 
-.content {
+.page {
   display: flex;
```
Comment (MauserBitfly) [inline frontend/pages/register.vue:142]: like for the login, the `flex` props are not necessary here.

File: frontend/pages/login.vue [hunk 2]
```diff
@@ -34,104 +34,132 @@ const canSubmit = computed(() => email.value && password.value && !Object.keys(e
...
-        @include fonts.small_text;
-        margin-right: auto;
+        display: flex;
+        flex-direction: column;
+        height: 100px;
+        .label {
+          display: flex;
+          flex-direction: row;
+          margin-bottom: 12px;
+          .right-cell {
```
Comment (MauserBitfly) [inline frontend/pages/login.vue:137]: According to Figma the forgot password link should have the small font size, like the register link

**Review by MauserBitfly — COMMENTED** [review]
Looks good, made some small remarks. 

While checking the register/login forms I realized that some accessibility things need to be changed. (was already wrong before) We will tackle this in a future PR, once Marcel's Form components are ready that handle that.

File: frontend/components/bc/header/MainHeader.vue [hunk 3]
```diff
@@ -145,8 +157,58 @@ const userMenu = computed(() => {
 $mobileHeaderThreshold: 600px;
 $smallHeaderThreshold: 1024px;
 
+.minimalistic {
+  position: relative;
+  display: flex;
+  width: 100%;
+  background-color: var(--dark-blue);
```
[the remark this answers is not in the document]
Comment (thib-wien) [reply frontend/components/bc/header/MainHeader.vue:164]: I followed the design :)

File: frontend/components/bc/header/MainHeader.vue [hunk 4]
```diff
@@ -145,8 +157,58 @@ const userMenu = computed(() => {
 $mobileHeaderThreshold: 600px;
 $smallHeaderThreshold: 1024px;
 
+.minimalistic {
```
[the remark this answers is not in the document]
Comment (thib-wien) [reply frontend/components/bc/header/MainHeader.vue:160]: OK, done.

File: frontend/components/bc/header/MainHeader.vue (hunk 3 above)
[the remark this answers is not in the document]
Comment (MauserBitfly) [reply frontend/components/bc/header/MainHeader.vue:164]: You are right, it was like that in the design (I didn't see) .

Talked to our designer: We change the logic -> The top header row should always be `var(--dark-blue);` to be consistent.

File: frontend/pages/login.vue (hunk 2 above)
Comment (thib-wien) [reply frontend/pages/login.vue:134]: OK, done :)

File: frontend/pages/register.vue (hunk 1 above)
Comment (thib-wien) [reply frontend/pages/register.vue]: Removed.

File: frontend/pages/login.vue [hunk 5]
```diff
@@ -34,104 +34,132 @@ const canSubmit = computed(() => email.value && password.value && !Object.keys(e
...
-.container {
-  position: relative;
-  width: 300px;
-  height: 240px;
-  margin: auto;
-  margin-top: 100px;
-  margin-bottom: 50px;
-  padding: var(--padding);
+.page {
+  display: flex;
```
[the remark this answers is not in the document]
Comment (thib-wien) [reply frontend/pages/login.vue]: Right! Removed.

File: frontend/components/bc/header/MainHeader.vue (hunk 4 above)
[the remark this answers is not in the document]
Comment (thib-wien) [reply frontend/components/bc/header/MainHeader.vue]: OK, done.

Note:
The `<BcLink>` containing the logo and the name of the website was present twice in the template and the the CSS was present twice with similar things + also differences.
So I created a `<HeaderLogo>` to remove those repetitions.

File: frontend/components/bc/header/MainHeader.vue (hunk 3 above)
[the remark this answers is not in the document]
Comment (thib-wien) [reply frontend/components/bc/header/MainHeader.vue]: OK, done.

**Review by MauserBitfly — APPROVED** [review]
Great job ;-)

**Comment by thib-wien** [comment]
> Great job ;-)

Thanks!
The next step will be to remove repetitions in the login and signup pages, but I think this is above my understanding, Marcel will maybe do it.

**Review by MauserBitfly — APPROVED** [review]

**Pull request merged**